### DISCUSSION TITLE
batch_size first dimension of the path coefficient

### DISCUSSION THREAD

**Discussion by amira123bd** [NVlabs/sionna#358] [Q&A]
The attenuation coefficient for each path is of dimension ([batch_size,num_rx,num,rx_ant,....]), I am wondering what does batch_size here represents exactly. 
Thank you in advance

**Comment by faycalaa** [comment]
Hi,

In general, the batch size corresponds to the number of independent realisations of the channel that are sampled.

If you are referring to the ray tracing module, the batch size dimension is added for compatibility with other modules, and its size is always one.
